src/NoFrixion.MoneyMoov/Models/PaymentRequests/PaymentRequest.cs

```diff
@@ -356,6 +356,11 @@ public PaymentMethodTypeEnum PaymentMethodTypes
     /// Minimal destination account details, if available.
     /// </summary>
     public PaymentAccount? DestinationAccount { get; set; }
+    
+    /// <summary>
+    /// Sandbox only. Optional. If set, the simulated Direct Debit settlement will be delayed by the specified number of seconds.
+    /// </summary>
+    public int? SandboxDirectDebitSettleDelayInSeconds { get; set; }

```

Comment:
Again my fault, coudl you rename to SandboxSettleDelayInSeconds so it's suitabel for pay by bank and DD.